modules/phoenix/src/main/ts/ephox/phoenix/util/Contiguous.ts

```diff
@@ -1,11 +1,13 @@
 import { Universe } from '@ephox/boss';
 import { Arr, Optional } from '@ephox/katamari';
 
+export interface Group<E> {
+  parent: E;
```

Comment:
A lot of these were just moved. I think I'd instead prefer to just log a ticket to go through and fix all interfaces so that they use readonly instead. Thoughts?

Reply:
I've fixed the one mentioned here, as I had to make a fix anyways. We should maybe still consider logging a JIRA to just do a massive sweep.